db/schema.rb

```diff
@@ -4,7 +4,7 @@
       primary_key :id
       Integer :confirmations_required, :default=>0, :null=>false
       Integer :last_keychain_id, :default=>0, :null=>false
-      String :pubkey, :size=>255
+      String :pubkey, :size=>255, :null=>false
```

Comment:
Nope. It can be null, if gateway uses external address provider. This is changed in https://github.com/snitko/straight-server/blob/master/db/migrations/014_pubkey_null_address_provider_not_null.rb , but schema.rb was not updated for some reason, so I edited it manually.